### DISCUSSION TITLE
Any Tool , Autonomously Manage Azure Cost Portfolio like ProsperOps does for AWS

### DISCUSSION THREAD

**Discussion by sumantaa** [microsoft/finops-toolkit#518] [Ideas]
If any expert in coding , create source code to deploy in Azure , that will dynamically manage the commitment portfolio. then it will be great asset for cost reduction.

**Comment by flanakin** [comment]
We've definitely talked about this. Are there any specific things you're looking for? Some will be easier than others so maybe we could start with small solutions to help make a dent.
